tensorboard/plugins/mesh_visualizer/test_utils.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Test utils for mesh plugin tests."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import google_type_annotations
+from __future__ import print_function
+
+import collections
+import json
+import threading
+import numpy as np
+import tensorflow as tf
+from google3.third_party.tensorboard.compat.proto import event_pb2
+from google3.third_party.tensorboard.compat.proto import graph_pb2
+from google3.third_party.tensorboard.compat.proto import meta_graph_pb2
+from google3.third_party.tensorboard.compat.proto import summary_pb2
+from google3.third_party.tensorboard.util import tb_logging
+
+Mesh = collections.namedtuple('Mesh', ('vertices', 'faces', 'colors'))
+logger = tb_logging.get_logger()
+
+
+# NOTE: copy FileWriter and FileWriterCache from tensorboard test_util.py
+# until this plugin start live in TensorBoard github repo.
+class FileWriter(tf.compat.v1.summary.FileWriter):
```

Comment:
I might be missing something but I am quite surprised that you have to implement FileWriter and FileWriterCache. In TensorFlow v2 + Summary v2 (and in eager), I believe you don't really have to do this.

Perhaps @nfelt can chime in here.

Reply:
I "borrowed" it from your test_utils since I couldn't really reference that file. I can either change it to use yours now (on github) or can do something else if it is preferred.

Reply:
switched to test_utils